cli/cmd/cluster_test.go

```diff
@@ -566,3 +570,77 @@ stderr line three`
 		})
 	})
 })
+
+var _ = Describe("GetClusterDataAssertOnCluster()", func() {
+	header := []string{"contentid", "hostname", "datadir"}
+	coordinatorDataDir := "./coordinator_data/gpseg-1"
+	segOneDataDir := "./seg_data/gpseg0"
+	segTwoDataDir := "./seg_data/gpseg1"
+	coordinator := []driver.Value{"-1", "cdw", coordinatorDataDir}
+	localSegOne := []driver.Value{"0", "sdw1", segOneDataDir}
+	localSegTwo := []driver.Value{"1", "sdw2", segTwoDataDir}
+
+	var (
+		connection *dbconn.DBConn
+		mock       sqlmock.Sqlmock
+	)
+
+	BeforeEach(func() {
+		connection, mock = testhelper.CreateAndConnectMockDB(1)
+	})
+
+	Context("When command is running on the coordinator", func() {
+		BeforeEach(func() {
+			_ = os.MkdirAll(coordinatorDataDir, 0777)
+		})
+
+		AfterEach(func() {
+			_ = os.RemoveAll("./coordinator_data")
```

Comment:
```suggestion
			_ = os.RemoveAll(testDir)
```
seems that was the value for the base test directory, maybe have it as a separate variable

Reply:
I changed the name of  these test directories. Since the following tests require to create/delete directories which may not  be allowed on root directory, I still use the relative paths for the testing